Sources/Testing/Testing.docc/Traits.md

```diff
@@ -14,24 +14,52 @@ Add traits to tests to annotate them or customize their behavior.
 
 ## Overview
 
-This article describes the ``Trait``, ``TestTrait``, and ``SuiteTrait``
-protocols, lists the traits provided by the testing library, and lists functions
-that can be used to create them.
-
-The ``Trait``, ``TestTrait``, and ``SuiteTrait`` protocols are used to define
-types that customize the behavior of test functions and test suites.
+Pass built-in traits to test functions or suite types to comment, categorize, 
+classify, and modify runtime behaviors. You can also use the ``Trait``, ``TestTrait``, 
+and ``SuiteTrait`` protocols to create your own types that that customize the 
+behavior of test functions.
 
 ## Topics
 
+### Customizing runtime behaviors
+
 - <doc:EnablingAndDisabling>
+- <doc:LimitingExecutionTime>
+- <doc:Parallelization>
+- ``Trait/enabled(if:_:fileID:filePath:line:column:)``
+- ``Trait/enabled(_:fileID:filePath:line:column:_:)``
+- ``Trait/disabled(_:fileID:filePath:line:column:)``
+- ``Trait/disabled(if:_:fileID:filePath:line:column:)``
+- ``Trait/disabled(_:fileID:filePath:line:column:_:)``
+- ``Trait/timeLimit(_:)``
+
+<!--
+HIDDEN: .serial is experimental SPI pending feature review.
+### Running tests serially or in parallel
+- ``SerialTrait``
+ -->
+
+### Annotating tests
+
 - <doc:AddingTags>
 - <doc:AddingComments>
 - <doc:AssociatingBugs>
-- <doc:LimitingExecutionTime>
-- <doc:Parallelization>
+- <doc:BugIdentifiers>
+- ``Tag()``
```

Comment:
Should we consider grouping this a bit differently so that the link to the `@Tag` macro is near the link to AddingTags.md rather than having comments/bugs in the middle? (I recognize they're sorted alphabetically.)

Reply:
I think the current curation works. The "Adding tags..." article should include a link to the `Tag` macro reference page, so floating that below is OK. The ordering here, however, should reflect an order of preference or expectation on the part of the developer. Do you expect them to add tags before comments or bugs? Or would it be comments, tags, bugs? That should influence the ordering in the curation.